src/code-snippet/code-snippet.stories.ts

```diff
@@ -0,0 +1,69 @@
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+
+import { CodeSnippetModule } from "..";
+
+
+const code = `import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs, boolean } from "@storybook/addon-knobs/angular";
+
+import { CodeSnippetModule } from "..";
+
+storiesOf("CodeSnippet", module).addDecorator(
+	moduleMetadata({
+		imports: [CodeSnippetModule]
+	})
+)
+	.addDecorator(withKnobs)
+	.add("Basic", () => ({
+		template: \`<ibm-code-snippet>code</ibm-code-snippet>\`,
+		props: { // there's more
+			// disabled: boolean("disabled", false)
+		}
+	}));`;
+
+const lessCode = `import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs, boolean } from "@storybook/addon-knobs/angular";
+
+import { CodeSnippetModule } from "..";
+
+storiesOf("CodeSnippet", module).addDecorator(
```

Comment:
```suggestion
storiesOf("Code Snippet", module).addDecorator(
```

Reply:
This is also inconsistent in the library. Question becomes do we want class names or unrelated titles?

@cal-smith any preference?

Reply:
React storybook is also inconsistent ... I'd go with related titles ... so "Code Snippet" would work